### DISCUSSION TITLE
24px icons no longer supported?

### DISCUSSION THREAD

**Discussion by Suzie97** [elementary/docs#104] [Q&A] [answered]
Hey, I've been working on an app for AppCenter and noticed that the docs don't mention the need for 24px icons. But other elementary designed apps do have a 24px icon. 

Is this a new change or a typo in the docs?

**Comment by danirabbit** [comment, marked as answer]
Yes, we were trying to drop the need for 24px icons to make things a little easier for developers but we're actually still using them in notifications so this documentation needs to be amended to re-include 24px

**Comment by Suzie97** [comment]
Can the docs mention the places where these icons are used? Btw, I did read about it on the official blog. So maybe a link to that blog post?

**Reply by danirabbit** [reply]
I've proposed a branch that includes a table listing places where icons are used https://github.com/elementary/docs/pull/110
